js/src/main/scala/Main.scala

```diff
@@ -141,69 +125,54 @@ object Main {
         val sb = new StringBuilder()
         sb ++= "<font color=\"red\">Code generation crashed:</font>"
         sb ++= htmlLineBreak + e.getMessage
+        // sb ++= htmlLineBreak + ((e.getStackTrace) mkString htmlLineBreak)
         sb ++= htmlLineBreak
         sb ++= htmlLineBreak
         L(sb.toString)
     }
   }
-  
+
   // Execute the generated code.
   // We extract this function because there is some boilerplate code.
   // It returns a tuple of three items:
   // 1. results of definitions;
   // 2. results of expressions;
   // 3. error message (if has).
   @SuppressWarnings(Array("org.wartremover.warts.AsInstanceOf"))
-  private def executeCode(
-      code: Str
-  ): (collection.mutable.HashMap[Str, Str], Ls[Str], Str) = {
-    // Collect evaluation results.
-    var defResults = new collection.mutable.HashMap[Str, Str]
-    var exprResults: Ls[Str] = Nil
-    var sb = new StringBuilder()
+  private def executeCode(code: Str): Either[Str, Ls[Str]] = {
     try {
-      // I know this is very hacky but using `asIntanceOf` is painful.
-      // TODO: pretty print JavaScript values in a less hacky way.
-      // Maybe I can add a global function for reporting results.
-      val results = js.eval(code).asInstanceOf[js.Dictionary[js.Any]]
-      results.get("defs").foreach { defs =>
-        defs.asInstanceOf[js.Dictionary[Str]] foreach { case (key, value) =>
-          defResults += key -> value
-        }
-      }
-      results.get("exprs").foreach { exprs =>
-        exprResults = exprs.asInstanceOf[js.Array[Str]].toList
-      }
+      R(js.eval(code).asInstanceOf[js.Array[Str]].toList)
     } catch {
       case e: Throwable =>
-        sb ++= "<font color=\"red\">Runtime error occurred:</font>"
-        sb ++= htmlLineBreak + e.getMessage
-        sb ++= htmlLineBreak
-        sb ++= htmlLineBreak
+        val errorBuilder = new StringBuilder()
+        errorBuilder ++= "<font color=\"red\">Runtime error occurred:</font>"
+        errorBuilder ++= htmlLineBreak + e.getMessage
+        errorBuilder ++= htmlLineBreak
+        errorBuilder ++= htmlLineBreak
+        L(errorBuilder.toString)
     }
-    (defResults, exprResults, sb.toString)
   }
-  
```

Comment:
Did you find a way to configure scalafmt so it does not do this? I'd like you not to commit changes like this. Removing indentation is fine if the whole team uses an automatic formatter, but for me it's a pain.

Reply:
These annoying changes were reverted. I searched a while in their document and found nothing. So I opened an issue requesting their help.

Reply:
They said they won’t implement this feature. So ignoring this file should work.

Reply:
Can you link the issue here please?

Reply:
Link: scalameta/scalafmt#3006